doc/citing.rst

```diff
@@ -12,4 +12,5 @@ version. If you are citing HoloSpy because you have used it to process data,
 please use the DOI of the specific version that you have employed. You can
 find it by clicking on the DOI badge below:
 
-To be updated after first release.
+.. image:: https://zenodo.org/badge/doi/10.5281/zenodo.10252344.svg
```

Comment:
Similarly as in https://github.com/hyperspy/holospy/pull/27, maybe this page can be removed since this is already in in the `intro.rst` page?

Reply:
Indeed, it is actually not displayed anywhere and must have been forgotten in a previous PR when the information was placed in `intro.rst`.